framework/test/unit/modules/interoperability/mainchain/internal_method.spec.ts

```diff
@@ -144,5 +144,21 @@ describe('Mainchain interoperability internal method', () => {
 
 			expect(isLive).toBe(false);
 		});
+
+		// TODO [DONE]: A test is missing for the case where the chain account exists, status is ACTIVE and liveness requirement IS fulfilled.
```

Comment:
```suggestion
		// TODO [DONE]: A test is missing for the case where the chain account exists, status is ACTIVE and liveness requirement is fulfilled.
```

Reply:
This is a TODO line, will be removed soon